### ISSUE TITLE
Looking for a Job: 

### ISSUE THREAD

**Issue by swifstudios** [wet-boew/wet-boew#8653]
Hello,

What generates the following timecode? I can't imagine having to enter all this code manually. Is there a software that would export this HTML code from the transcribed video?

<h2>Transcript</h2>
<p class="wb-vd"><strong>(Animated pen draws a red line that leads into the text Looking for A Job)</strong></p>
<p class="wb-vd"><strong>(Cut to a medium shot of the Host. LINE DRAWING GRAPHIC: Stick person appears and waves.)</strong></p>
<p><span class="wb-tmtxt" data-begin="2.50s" data-dur="3.84s">Hi, my name is Eric, and I&#39;m a Service Canada employee.</span></p>
<p><span class="wb-tmtxt" data-begin="6.34s" data-dur="1.87s">You may have heard the old saying that, </span>
<span class="wb-tmtxt" data-begin="8.21s" data-dur="1.84s">when you&#39;re out of work, finding a job is your </span>

**Comment by juleskuehn** [comment]
What are the formats of data you have currently for the transcribed video? TTML?

**Comment by duboisp** [comment]
@swifstudios yes, when using the wet-boew HTML inline markup for close caption and transcript that is manual and I am not aware of a wet-boew tool that would ease its codification.

But, if using TTML markup language is an option for you to code your close caption and transcript for your video, then there is might be some tools out there that you can use. Currently, I have no specific tool to recommend you.
